modules/kubernetes-addons/kube-prometheus-stack/values.yaml

```diff
@@ -0,0 +1,20 @@
+## Create default rules for monitoring the cluster
```

Comment:
Can we also disable alert manager or any other CRDs that is being deployed by this add-on which may not used by the users? User can enable through values.yaml if they want any of those

Reply:
I disabled the components that are unreachable in EKS but lean towards leaving the other resources at the upstream defaults. 
Mostly this is because the further we get from the defaults the more the user has to be aware of the adjustments we make. It could get confusing to understand why AlertManager/Thanos wasn't configured if using a values configuration from a pre-existing setup and we've disabled those components or CRDs.